core/ir/instr_api.h

```diff
@@ -2337,6 +2337,11 @@ DR_API
 instr_t *
 instr_create_pusha(void *drcontext);
 
+DR_API
+/** Returns whether \p instr loads mcontext base to some reg. */
```

Comment:
nit: I think this needs to explain what "mcontext" is and why this would be useful and where; plus it should include articles (s/mcontext/the mcontext/).